sound/soc/soc-dapm.c

```diff
@@ -3651,6 +3651,10 @@ snd_soc_dapm_new_control_unlocked(struct snd_soc_dapm_context *dapm,
 	case snd_soc_dapm_dac:
 	case snd_soc_dapm_aif_in:
 	case snd_soc_dapm_pga:
+	case snd_soc_dapm_buffer:
+	case snd_soc_dapm_scheduler:
+	case snd_soc_dapm_src:
+	case snd_soc_dapm_effect:
```

Comment:
Meh. this isn't the same order as definitions and it's missing asrc/encoder/decoder.